src/autocomplete.ts

```diff
@@ -121,7 +172,24 @@ export default class Autocomplete {
       .then(html => {
         this.results.innerHTML = html
         this.identifyOptions()
-        const hasResults = !!this.results.querySelector('[role="option"]')
+        const allNewOptions = this.results.querySelectorAll('[role="option"]')
+        const hasResults = !!allNewOptions.length
+        const numOptions = allNewOptions.length
+
+        // add active descendant attribute to the input so that it's clear what Enter will do
+        if (this.autoselectEnabled) {
+          const [firstOption] = [...allNewOptions]
+          const firstOptionValue = firstOption?.textContent
+          const firstOptionId = firstOption?.id
+
+          if (firstOption) {
+            this.input.setAttribute('aria-activedescendant', firstOptionId)
```

Comment:
Do we have any visual indication that the item is activedescendant at this point?

Reply:
Not yet. I focused on screenreader feedback on this guy, and styling is coming. Next up is "clearable" interactions though.